lib/node-utils.js

```diff
@@ -407,28 +407,44 @@ function hasStaticModifierFlag(node) {
  * Finds the next token based on the previous one and its parent
  * @param {TSToken} previousToken The previous TSToken
  * @param {TSNode} parent The parent TSNode
+ * @param {ts.SourceFileLike} ast The TS AST
  * @returns {TSToken} the next TSToken
  */
-function findNextToken(previousToken, parent) {
-  /**
-   * TODO: Remove dependency on private TypeScript method
-   */
-  return ts.findNextToken(previousToken, parent);
```

Comment:
We should just make this public.

Reply:
Note that the implementation I wrote differs very slightly (but importantly) from our implementation, so we should fix that too. But that's why I had to change it.